index.html

```diff
@@ -1251,12 +1251,8 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/lyla-advice.html">Lyla's Advice</a><br>
       <a href="advice/SueWigand-advice.html">Sue Wigand's Advice</a><br>
       <a href="advice/jeanette-h-advice.html">Jeanette H's Advice</a><br>
```

Comment:
The six links below have been removed from the index.html file. Once you've added them back you can add and commit the change and then push your code to GitHub.